release/pkg/manager/calico/manager.go

```diff
@@ -477,6 +468,9 @@ func (r *CalicoManager) releasePrereqs() error {
 
 // Prerequisites specific to publishing a release.
 func (r *CalicoManager) publishPrereqs() error {
+	if r.isHashRelease {
+		return nil
```

Comment:
Do we not have any other hashrelease prereqs? e.g., if it already exists on the server?

I think we're checking that elsewhere, but why not here? Seems better to consolidate into one location. As it stands, it's not obvious when something belongs in the CalicoManager vs. when it just goes into main.go.

Reply:
I can add the check here. However it is in main so that operator images do not get pushed if the hashrelease already exist